docs/sdkx-cocos2dx/support-tools.mdx

```diff
@@ -0,0 +1,143 @@
+---
+sidebar_position: 20
+title: Helpshift APIs
+description: "Helpshift provides a way to use each functional element of the SDK separately based on your requirements."
+---
+
+import {
+  Admonition,
+  Tabs,
+  TabItem,
+  Centered,
+  Image,
+  Intro,
+} from "@site/src/components/forDocs";
+
+# Helpshift APIs {#support-tools}
+
+<Intro>
+
+Helpshift provides a way to use each functional element of the SDK separately based on your requirements. You can call only specific screens from the SDK based on the functionality you want to expose in your app.
+
+</Intro>
+
+<Admonition type="info" title="Note">
+
+All the public APIs in the SDK should be called after initializing the SDK via [install API](/sdkx-cocos2dx/getting-started#initializing)
+
+</Admonition>
+
+## Integrating Contact Us & In App Messaging {#conversation-view}
+
+You can use the API call `HelpshiftCocos2dx::showConversation(configMap)` to allow a user to directly send feedback or start a new conversation without having to first view FAQs. Once, a user starts a new conversation, this API call will show the conversation screen. The conversation will continue until it is resolved or rejected by the agent.
+To configure SDK, you can pass various config options in a `cocos2d::ValueMap`. Checkout more config options [here](/sdkx-cocos2dx/sdk-configuration).
+
+<Centered width="major">
+
+![newConversation.png](/static/books/sdkx_android/show_conversation.png "newConversation.png")
+
+</Centered>
+
+Example :
+
+```cpp
+cocos2d::ValueMap getConfig() {
+    cocos2d::ValueMap configMap;
+
+    ValueVector tags;
```

Comment:
Don't think we should show code for adding tags here.

Reply:
@jaydeep-helpshift I added just to add something in config, Will try to make it generic.